saiph/inverse_transform.py

```diff
@@ -37,7 +38,10 @@ def inverse_transform(
     random_gen = np.random.default_rng(model.seed)
 
     # Check dimension size regarding N
-    n_dimensions = len(model.dummy_categorical) + len(model.original_continuous)
+    n_dimensions = get_number_of_dimensions_from_categorical_and_continuous(
+        dummy_categorical=model.dummy_categorical,
+        original_continuous=model.original_continuous,
+    )
```

Comment:
Note: I was first using `model: Model` as the only input, but as it's a `@DataClass`, I could not generate a factory for it.
So I went to use that workaround instead